src/test/regress/expected/subquery_view.out

```diff
@@ -559,9 +560,80 @@ DEBUG:  Plan XXX query after replacing subqueries and CTEs: SELECT user_id, user
        3 |       6 | Thu Nov 23 14:00:13.20013 2017  |          3 |       3 |       3 |
 (5 rows)
 
+CREATE TABLE reference_table (text_col text, int_col int);
+SELECT create_reference_table('reference_table');
+ create_reference_table
+---------------------------------------------------------------------
+
+(1 row)
+
 SET client_min_messages TO DEFAULT;
+EXPLAIN (COSTS OFF) WITH cte AS (
+  SELECT application_name AS text_col
+  FROM pg_stat_activity
+) SELECT * FROM reference_table JOIN cte USING (text_col);
+                                            QUERY PLAN
+---------------------------------------------------------------------
+ Custom Scan (Citus Adaptive)
+   ->  Distributed Subplan XXX_1
+         ->  Function Scan on pg_stat_get_activity s
+   ->  Distributed Subplan XXX_2
+         ->  Custom Scan (Citus Adaptive)
+               Task Count: 1
+               Tasks Shown: All
+               ->  Task
+                     Node: host=localhost port=xxxxx dbname=regression
+                     ->  Hash Left Join
+                           Hash Cond: (intermediate_result.usesysid = u.oid)
+                           ->  Hash Left Join
+                                 Hash Cond: (intermediate_result.datid = d.oid)
+                                 ->  Function Scan on read_intermediate_result intermediate_result
+                                 ->  Hash
+                                       ->  Seq Scan on pg_database d
+                           ->  Hash
```

Comment:
how stable is this plan?

Reply:
>(maybe push a few times)

I guess I couldn't understand this %100

Assuming below query, does that mean to wrap cte in SELECT in a few levels of subqueries to make sure that plan is stable ?
>EXPLAIN (COSTS OFF) WITH cte AS (
  SELECT application_name AS text_col
  FROM pg_stat_activity
) SELECT * FROM reference_table JOIN cte USING (text_col);

Reply:
sorry, I meant: it would be good to re-run CircleCI a few times to make sure the planner does not occasionally pick a different query plan for this join.